### DISCUSSION TITLE
How to remove animations in open brackets or functions?

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#2209] [Q&A] [answered]
Every time you enter bracket or function, a small animation starts to show in which bracket I am in. But is this possible to just disable it? I thought this was the issue with indent-blankline.nvim plugin configuration, but I don't see any animations. Or does it provide these animations out of the box?

**Comment by dpetka2001** [comment, marked as answer]
`mini.indentscope`. See its docs for detailed configuration or disable it if you don't like it.

**Comment by Demianeen** [comment]
@dpetka2001, that's helped a lot!
Why do we use two indent libraries though? I mean mini.indentscope and indent-blankline.nvim

**Reply by dpetka2001** [reply]
My guess would be for the visual aethetic. If I recall correctly indent-blankline didn't provide animation for the current indent scope at the time (maybe i don't remember correctly not sure). It was just personal preference for how it would look like.
